k-distribution/tests/regression-new/issue-567/test.k

```diff
@@ -0,0 +1,16 @@
+module TEST-COMMON
+  imports INT-SYNTAX
+  imports BOOL
+
+  syntax ShiftInst ::= Int "=" "shift" Int "," Int [hybrid, seqstrict(2,3)]
```

Comment:
What does `hybrid` mean? Just curious.

Reply:
hybrid introduces an axiom saying that if the term has been completely heated, then it is itself a KResult.